### PR TITLE
add libsass

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
this will require changes to `rubygem-sassc`, see https://github.com/theforeman/foreman-packaging/pull/10152

File: package_manifest.yaml [hunk 1]
```diff
@@ -246,6 +246,7 @@ ruby_core_packages:
 
 ruby_core_packages_tier1:
   hosts:
+    libsass: {}
```
Comment (evgeni) [inline package_manifest.yaml:249]: feels a bit weird to have it in `ruby_core_packages`, but then again, it's needed for ruby things…

**Comment by ekohl** [comment]
> this will require changes to `rubygem-sassc`, see #10152

Would it also move rubygem-sassc to a tier2 package?

File: packages/foreman/libsass/libsass.spec [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+%configure --disable-static
+make %{?_smp_mflags}
+
+
+%install
+make install DESTDIR=$RPM_BUILD_ROOT
+find $RPM_BUILD_ROOT -name '*.la' -exec rm -f {} ';'
+
+
+%ldconfig_scriptlets
```
Comment (ekohl) [inline packages/foreman/libsass/libsass.spec:53]: Is this supported in EL8? https://docs.fedoraproject.org/en-US/epel/epel-packaging/#shared_libraries phrasing implies it is, but it's not explicit.

**Comment by evgeni** [comment]
> > this will require changes to `rubygem-sassc`, see #10152
> 
> Would it also move rubygem-sassc to a tier2 package?

It already is (as it depends on ffi)

File: packages/foreman/libsass/libsass.spec (hunk 2 above)
Comment (evgeni) [reply packages/foreman/libsass/libsass.spec:53]: Kinda sure, yeah.

But will double check

**Review by ekohl — APPROVED** [review]
I'm assuming you're adding it on both EL 8 & 9 so we're consistent on all platforms, rather than only adding it with .so.1 on EL9 & use the system .so.0 on EL8.

:+1: assuming the macro does work on EL8.

File: packages/foreman/libsass/libsass.spec (hunk 2 above)
Comment (evgeni) [reply packages/foreman/libsass/libsass.spec:53]: Interestingly, on both EL9 and EL8 this ends up generating nothing:

```
[root@15b1f7780590 /]# rpm -q --scripts libsass
[root@15b1f7780590 /]# 
```

**Pull request merged**